src/libaktualizr/primary/CMakeLists.txt

```diff
@@ -17,5 +17,7 @@ add_aktualizr_test(NAME aktualizr SOURCES aktualizr_test.cc PROJECT_WORKING_DIRE
 add_dependencies(t_aktualizr uptane_repo_full_no_correlation_id)
 
 add_aktualizr_test(NAME reportqueue SOURCES reportqueue_test.cc PROJECT_WORKING_DIRECTORY)
+add_aktualizr_test(NAME emptytargets SOURCES empty_targets_test.cc PROJECT_WORKING_DIRECTORY
```

Comment:
Inconsequential but this source file is introduced in a later commit.

Reply:
Oops. If anything else comes up, I can fix that, but I'm not going to rebase just for that.